project_task_dependency/tests/test_project_task_dependency.py

```diff
@@ -71,11 +71,10 @@ def test_copy(self):
         self.assertEqual(task4.dependency_task_ids[0].id, self.task2.id)
 
     def test_regression_copy(self):
-        old_count = self.env["project.task.copy.map"].search_count(
+        self.env["project.task.copy.map"].search_count(
             [("old_task_id", "=", self.task4.id)]
         )
         self.task4.copy()
-        new_count = self.env["project.task.copy.map"].search_count(
+        self.env["project.task.copy.map"].search_count(
             [("old_task_id", "=", self.task4.id)]
         )
-        self.assertEqual(old_count, new_count)
```

Comment:
Why are you removing this assert? This gets orphaned with it

Reply:
My mistake, fixed!